sdcm/remote.py

```diff
@@ -225,6 +227,9 @@ def send_files(self, src, dst, delete_dst=False,  # pylint: disable=too-many-arg
             self.run(f'cp {src} {dst}')
 
 
+LOCALRUNNER = LocalCmdRunner()
```

Comment:
where this is used in this file ?

Reply:
never mind, I've got it... it being use in other files

Reply:
It's imported in other files. Why do we need many `LocalCmdRunner` instances?